apps/backend/src/database/database.module.ts

```diff
@@ -47,7 +47,7 @@ function sanitize(fields: string[], values?: string[]): string[] {
       } else {
         return value;
       }
-    }) || []
+    }) ?? []
```

Comment:
stuff like this is a false positive since it changes the behavior in a way we do not want.

we explicitly are looking for falsy on the left side || default value.  using ?? only does the default value when the left side is null or undefined.  many times we expect to see an empty string there which means that the default value is not put in instead and we have a bug on our hands.